client/disputes/index.tsx

```diff
@@ -241,58 +248,143 @@ export const DisputesList = (): JSX.Element => {
 
 	const downloadable = !! rows.length;
 
-	function onDownload() {
-		const title = __( 'Disputes', 'woocommerce-payments' );
+	const onDownload = async () => {
+		setIsDownloading( true );
+		const downloadType = totalRows > rows.length ? 'endpoint' : 'browser';
+		const userEmail = wcpaySettings.currentUserEmail;
 
-		const csvColumns = [
-			{
-				...headers[ 0 ],
-				label: __( 'Dispute Id', 'woocommerce-payments' ),
-			},
-			...headers.slice( 1 ),
-		];
+		if ( 'endpoint' === downloadType ) {
+			const {
```

Comment:
I think `type_is` should be `status_is` and `type_is_not` should be `status_is_not`.

As for the `search` filter, I'm not sure it is available on the disputes page? Couldn't find how to trigger it at least, maybe you know.. But I think we'll be able to remove this one as I can't see it being used in `getDisputesCSV` > `formatQueryFilters` anyway